resources/sdk/pureclouddotnet/templates/ApiClient.mustache

```diff
@@ -99,7 +101,29 @@ namespace {{packageName}}.Client
         /// Gets or sets the RestClient.
         /// </summary>
         /// <value>An instance of the RestClient</value>
-        public RestClient RestClient { get; set; }
+        public IRestClient RestClient { get; set; }
```

Comment:
Why was the type changed here? `RestClient` -> `IRestClient`

Reply:
`Moq` (and standard mocking frameworks) are creating proxy classes and it can only mock interfaces and virtual methods.

Reply:
Ah, got it. The test file was collapsed when I was searching for usages so I missed where it was being used that way.